docs/API/metadata.md

```diff
@@ -120,5 +120,11 @@ distinct values for the same keys, or use completely different keys, and can
 retrieve those values later. A client authenticated for another user has
 its own comletely unique `user` namespace.
 
+The `user` metadata namespace behaves as a user-specific sub-resource under the
+dataset. Any authenticated client has UPDATE and DELETE access to this private
+sub-resource as long as the client has READ access to the dataset. See
```

Comment:
There's an interesting foible here:  if a user un-publishes a dataset to which a second user has attached `user` metadata, then the second user will lose access to his/her metadata even though it presumably will continue to _exist_ as long as the dataset does.  (And, if the dataset is re-published, then the second user's metadata will become available again.)

I'm not saying that this is good or bad, I'm just pointing out what will likely fall out of this.  🙃

Reply:
Yeah, it's true, and I have no idea whether it's "good" or "bad". If the dataset transitions to PRIVATE (and the owner of the `user` metadata doesn't have `ADMIN` role), and it *remains* private permanently, then the data is invisible. But I suspect that "unpublish" is probably a rare event? And maybe it's a useful feature that "publish" will *"restore"* the data?

Don't know; but I added a note in the new resource section.